spec/models/task_spec.rb

```diff
@@ -1,5 +1,43 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validation' do
+    it 'is valid with all attributes' do
+      user = User.create(email: "user@example.com", password: "password")
+      task = user.tasks.new(title: "aaa", content: "aaa", status: :todo)
+      expect(task.valid?).to be(true)
```

Comment:
`be_valid`マッチャを使うと`task.valid?`と記載する必要が無くなります！
```suggestion
      expect(task).to be_valid
```

Reply:
7860184